src/sentry/receivers/features.py

```diff
@@ -146,11 +145,25 @@ def record_issue_assigned(project, group, user, **kwargs):
     )
 
 
-@issue_resolved_in_release.connect(weak=False)
-def record_issue_resolved_in_release(project, group, user, resolution_type, **kwargs):
-    FeatureAdoption.objects.record(
-        organization_id=project.organization_id, feature_slug="resolved_in_release", complete=True
-    )
+@issue_resolved.connect(weak=False)
+def record_issue_resolved(organization_id, project, group, user, resolution_type, **kwargs):
+    """ There are three main types of ways to resolve issues
+        1) via a release (the next release, or other (either current or another version which are indistinguishable))
+        2) via commit
+        3) now
+    """
+    if resolution_type in ('in_next_release', 'in_release'):
+        data_resolution_type = resolution_type
+        FeatureAdoption.objects.record(
+            organization_id=organization_id, feature_slug="resolved_in_release", complete=True
+        )
+    elif resolution_type == 'in_commit':
+        data_resolution_type = 'with_commit'
```

Comment:
I still don't understand what's going on here. in_commit is different from with_commit?

Reply:
`with_commit` is the label we send to analytics event but `in_commit` is what is sent here https://github.com/getsentry/sentry/pull/12412/files#diff-9577ad7c55d768a4da4a28e48d27c522R677 so I was trying to be consistent. Alternatively I can re-define `res_type_str` from `in_commit` to `with_commit` at that point. Is this what you were thinking?